content/04.results.md

```diff
@@ -2,8 +2,22 @@
 
 ### Comparing bioRxiv to PubMed Central
 
-#### Global View
-1. Create a treemap visualization of top X terms that are different between bioRxiv and PubMed Central (based on odds ratio)
+#### Global Comparison of bioRxiv and PubMed Central 
+
+![
+BioRxiv has more category specific terms whereas PubMed Central has more patient oriented articles.
+The plot on the left (A) is a point range plot of the odds ratio with respect to bioRxiv.
+Values greater than one indicate a high association with bioRxiv whereas values less than one indicate high association with PubMed Central.
+The dotted line provides a breaking point between both categories.
+The plot on the right (B) is a barchart of token frequency appearing in bioRxiv and PMC respectively.
+](https://raw.githubusercontent.com/greenelab/annorxiver/e32661b855cc6622cc1138fcd1606ef3500f0ff9/biorxiv/corpora_comparison/output/figures/biorxiv_vs_pubmed_central.png){#fig:biorxiv_pmc_global_comparison}
+
+The linguistic style of preprints can be compellingly different than published peer reviewed articles.
+We quantified the difference between preprints in biorxiv to published articles in PubMed Central (PMC) by calculating the odds ratio on tokens appearing in both repositoies respectively.
+Tokens with a high odds ratio are topic-specific words such as: "neuron", "genome", "RNA" and "network" (Figure {@fig:biorxiv_pmc_global_comparison}).
```

Comment:
Can you try this starting framing and then follow-on with the low odds ratio and close with an interpretation of your findings? Do you think this is how people are writing, typesetting, or the topic areas included/absent from bioRxiv?